app/controllers/spaces_controller.rb

```diff
@@ -20,6 +20,9 @@ def update
   end
 
   def destroy
+    @space = Space.find(params[:id])
+    @space.destroy
+    redirect_to space_path(@space)
```

Comment:
You cannot redirect to a space you've just destroyed; you likely mean `spaces_path`